### PR TITLE
QC report updates

### PR DESCRIPTION
**Pull request by shawnpg**
Update QC thresholds, update histogram style, and rename QC script to something more appropriate

### CODE REVIEW

**Comment by maximilianpress** [comment]
Looks like Travis failed. Changes to API probably responsible at a guess?

--> yes, it looks like filename change led to broken import in tests.

File: collateral/thresholds.json [hunk 1]
```diff
@@ -0,0 +1,21 @@
+{
+	"genome" :	{
+				"index"				:	"0",
```
Comment (bnelsj) [inline collateral/thresholds.json]: I'd prefer yaml here for sane type inference (don't need to write floats as strings then read strings as floats)

File: hic_qc.py [hunk 2]
```diff
@@ -628,39 +724,19 @@ def pass_judgement(self):
...
-            long_contacts = False
-            long_floor = False
-            useful_contacts = False
-            many_zero_pairs = False
-            many_many_zero_pairs = False
-            high_dupe = False
-
-        low_contiguity = self.N50 < 100000
+        # We are stricter on wanting a low number of dupes when it looks like we are only looking at a QC amount of sequencing (<10M read pairs)
+        allowed_dupe_percentage = 1.0 if self.stats['total_read_pairs'] > 1e7 else 0.5
```
Comment (bnelsj) [inline hic_qc.py]: It might be good to set this as a property since it's calculated by multiple functions

File: hic_qc.py [hunk 3]
```diff
@@ -628,39 +724,19 @@ def pass_judgement(self):
...
-        bad = False
-        if low_contiguity:
-            if many_many_zero_pairs or high_dupe:
-                bad = True
-        else:
-            if many_zero_pairs or high_dupe or not long_floor:
-                bad = True
-        self.judge_good = good
-        self.judge_bad = bad
+        self.judge_good = self.long_contacts and self.useful_contacts and self.same_strand_hq
```
Comment (bnelsj) [inline hic_qc.py]: Nice, this is way less mysterious

File: hic_qc.py [hunk 4]
```diff
@@ -917,6 +1009,10 @@ def parse_args():
     parser.add_argument('--mq_stats', nargs='+', default=[0, 1, 10, 20, 30, 40], help='List of min MQ scores to calculate RP stats for (Default: %(default)s)')
     parser.add_argument('--edist_stats', nargs='+', default=[100, 10, 5, 3, 1, 0], help='List of max edist scores to calculate RP stats for (Default: %(default)s)')
     parser.add_argument('--version', action='version', version=__version__)
+    parser.add_argument('--thresholds', default='./collateral/thresholds.json',
```
Comment (bnelsj) [inline hic_qc.py]: This should be determined relative to the python script using the `__file__` property

File: collateral/thresholds.json (hunk 1 above)
Comment (shawnpg) [reply collateral/thresholds.json]: Tomato, tomahto :) What I would like to do is make it support either YAML or JSON, but I don't think that's important now

File: hic_qc.py (hunk 2 above)
Comment (shawnpg) [reply hic_qc.py]: Good idea. Did that

File: hic_qc.py (hunk 3 above)
Comment (shawnpg) [reply hic_qc.py]: 👍

File: hic_qc.py (hunk 4 above)
Comment (shawnpg) [reply hic_qc.py]: Done

**Pull request merged**